web/src/components/software/SoftwarePage.tsx

```diff
@@ -121,25 +127,23 @@ function SoftwarePage(): React.ReactNode {
   const ReloadSection = (): React.ReactNode => (
     // TRANSLATORS: title for an error message box, at least one repository could not be loaded
     <Alert variant="danger" isInline title={_("Repository load failed")}>
-      {/* TRANSLATORS: error details followed by a "Try again" link*/}
-      {_(
-        "Some installation repositories could not be loaded. The system cannot be installed without them.",
-      )}{" "}
       {loading ? (
         <>
-          {" "}
-          <Spinner size="md" />
+          <Spinner size="md" /> {_("Loading the installation repositories...")}
         </>
       ) : (
-        <Button variant="link" isInline onClick={startProbing}>
-          {/* TRANSLATORS: link for retrying failed repository load */}
-          {_("Try again")}
-        </Button>
+        <>
+          {errorMsg}{" "}
+          <Button variant="link" isInline onClick={startProbing}>
+            {/* TRANSLATORS: link for retrying failed repository load */}
+            {_("Try again")}
+          </Button>
+        </>
```

Comment:
Sorry, I forgot to mention in yestereday's review. Please, move this component outside of the component is using it. As far as I can see yo just need to give it an `onReload` prop 

```jsx
<ReloadSection onReload={startProbing} />
```

Reason for moving it outside: https://react.dev/learn/preserving-and-resetting-state.

We have had issues with that, for example https://github.com/agama-project/agama/pull/1550#issuecomment-2329451247

Reply:
Thanks a lot about this, I didn't know it!